### PR TITLE
DW-103: reverse proxy for webapp

### PR DESCRIPTION
**Pull request by cbernat**
Added signing for webapp in reverse proxy.
- Exposed 443 port for proxy
- Added certificates as secrets in configuration
- Modified conf for webapp an playground for local testing

### CODE REVIEW

File: sites-proxy/conf.d/play.fromdoppler.com.conf [hunk 1]
```diff
@@ -1,3 +1,4 @@
+error_log /var/log/nginx/error.log warn;
```
Comment (andresmoschini) [inline sites-proxy/conf.d/play.fromdoppler.com.conf]: What is this?

**Comment by andresmoschini** [comment]
👏 

Remember also adding the related PR here: https://github.com/MakingSense/doppler-swarm-devops

File: sites-proxy/conf.d/play.fromdoppler.com.conf (hunk 1 above)
Comment (cbernat) [reply sites-proxy/conf.d/play.fromdoppler.com.conf]: This is not supposed to be here. I will delete it. Thanks!

**Comment by cbernat** [comment]
```
return 301 https://$server_name$request_uri;
```
In conf files seems to prevent the service to load, and it's neccesary, I'll research into this.

**Comment by cbernat** [comment]
> ```
> return 301 https://$server_name$request_uri;
> ```
> 
> In conf files seems to prevent the service to load, and it's neccesary, I'll research into this.



> ```
> return 301 https://$server_name$request_uri;
> ```
> 
> In conf files seems to prevent the service to load, and it's neccesary, I'll research into this.

Tested this throughly and there are no issues, redirect is working ok.

**Comment by cbernat** [comment]
> 👏
> 
> Remember also adding the related PR here: https://github.com/MakingSense/doppler-swarm-devops
Added in other repository
https://github.com/MakingSense/doppler-swarm-devops/pull/1

**Pull request merged**